src/workerd/api/streams/queue.h

```diff
@@ -366,7 +366,8 @@ class ConsumerImpl final {
   void error(jsg::Lock& js, jsg::Value reason) {
     // If we are already closed or errored, then we do nothing here.
     // The new error doesn't matter.
-    KJ_IF_MAYBE(ready, state.template tryGet<Ready>()) {
+    KJ_IF_SOME(ready, state.template tryGet<Ready>()) {
+      (void)ready;  // Unused
```

Comment:
Just flagging this unused in case as it showed up as a compiler warning.

Reply:
For these, I would suggest switching the check to `if (state.template tryGet<Ready>() != kj::none) {` to avoid this entirely